doc/lua_api.txt

```diff
@@ -5695,13 +5714,13 @@ Item handling
         * `width`: 0-3, 0 means shapeless recipe
         * `items`: indexed [1-9] table with recipe items
         * `output`: string with item name and quantity
-    * Example query for `"default:gold_ingot"` will return table:
+    * Example result for `"default:gold_ingot"` with two recipes:
 
           {
               [1]={method = "cooking", width = 3, output = "default:gold_ingot",
-              items = {1 = "default:gold_lump"}},
+              items = {[1] = "default:gold_lump"}},
               [2]={method = "normal", width = 1, output = "default:gold_ingot 9",
-              items = {1 = "default:goldblock"}}
+              items = {[1] = "default:goldblock"}}
           }
```

Comment:
Suggestion: width is 0 for shapeless recipes and explicit indices are not needed here. This version is longer but easier to read.

```
              {
                  method = "cooking", width = 0,
                  output = "default:gold_ingot",
                  items = {"default:gold_lump"}
              },
              {
                  method = "normal", width = 0,
                  output = "default:gold_ingot 9",
                  items = {"default:goldblock"}
              }
```

Reply:
Ah so the example was not even correct?

Reply:
Yes, multi-input cooking and hence width > 0 is currently not possible for the `"cooking"` method.
`"normal"` could have width > 0, but requires nested tables for the registration.

Reply:
Just checked, here's the real result in MTG:
```lua
{
	{
		output = "default:gold_ingot",
		items = {
			"keys:skeleton_key"
		},
		method = "cooking",
		width = 5,
		type = "cooking"
	},
	{
		output = "default:gold_ingot",
		items = {
			"keys:key"
		},
		method = "cooking",
		width = 5,
		type = "cooking"
	},
	{
		output = "default:gold_ingot",
		items = {
			"default:gold_lump"
		},
		method = "cooking",
		width = 3,
		type = "cooking"
	},
	{
		output = "default:gold_ingot 9",
		items = {
			"default:goldblock"
		},
		method = "normal",
		width = 1,
		type = "normal"
	}
}
```

Reply:
I see. `width` originates from `cooktime`, thanks to `CraftDefinitionCooking::getInput`.
Meanwhile the `cooktime` parameter is not documented at all.

Reply:
wat :thinking: